{"uuid": "f411292f-78a4-4715-9027-1e0375411300", "children": ["9a294374-773b-4b4a-bc7c-894944b7a219", "e4b7358c-a4ff-4198-9138-5f3470fc200f", "5707c307-e2c0-421b-8a53-35bf21576c08", "ec796e52-5deb-4383-ab43-86ca5fd226e8", "3d01810c-122e-48dc-88d8-e8f6403e1bc2", "ea09e203-f786-4522-82b6-ffb84209d267", "e24c42db-e059-4bde-a5e7-928c86b1d128"], "befores": [{"name": "setup_browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: <html>\r\n<head><title>401 Authorization Required</title></head>\r\n<body bgcolor=\"white\">\r\n<center><h1>401 Authorization Required</h1></center>\r\n<hr><center>nginx/1.14.2</center>\r\n</body>\r\n</html>\r\n\n\n", "trace": "  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/conftest.py\", line 42, in setup_browser\n    driver = webdriver.Remote(\n             ^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 209, in check_response\n    raise exception_class(value)\n"}, "start": 1685198081185, "stop": 1685198081504}], "start": 1685198081185, "stop": 1685198081640}